client/src/services/releases.ts

```diff
@@ -201,14 +198,18 @@ async function downloadFile(
 
   console.warn('Downloading file of', totalBytes, 'bytes size from', url, 'to', destFilePath)
 
+  // The fetch API uses the ReadableStream from 'stream/web'
+  // Convert it the the native one
+  const bodyStream = stream.Readable.fromWeb(res.body)
+
   let readBytes = 0
-  res.body.on('data', (chunk: Buffer) => {
+  bodyStream.on('data', (chunk: Buffer) => {
```

Comment:
How can we test this code? I guess a clean install would work. Did you try it?

Reply:
Yes

Reply:
Do you happen to remember the local path I have to clean out to trigger a redownload? Or perhaps I can just run the command again?

Reply:
On Windows it's `C:\Users\<username>\AppData\Roaming\Code\User\globalStorage\flix.flix\flix.jar`